frontend/src/pages/modelServing/screens/global/InferenceServiceStatus.tsx

```diff
@@ -109,7 +109,7 @@ const InferenceServiceStatus: React.FC<InferenceServiceStatusProps> = ({
     <Popover
       data-testid="model-status-tooltip"
       position="top"
-      bodyContent={bodyContent}
+      bodyContent={<div style={{ maxHeight: '65vh', overflow: 'auto' }}>{bodyContent}</div>}
```

Comment:
@ashley-o0o is there a specific reason to use 65% of the available height? Why not allow the popover to consume the most amount of available vertical space?

Reply:
@christianvogt When I used 95%, it still extended beyond the window on my laptop. Obviously this might change depending on the monitor, however, visually it looked better to me to have the popover overall smaller as opposed to using most of the screen. I can change it to 95% though if you believe that would be best.

Reply:
@ashley-o0o i wonder if that has to do with the content being updated dynamically after the popover has already positioned itself and then the new larger content doesn't trigger a revaluation of the popover position. If that's the case then 65% can still cause the same issue. I'll test this out.